replication/parser.go

```diff
@@ -100,7 +100,10 @@ func (p *BinlogParser) ParseReader(r io.Reader, onEvent OnEventFunc) error {
 		var e Event
 		e, err = p.parseEvent(h, data)
 		if err != nil {
-			break
+			if _, ok := err.(missingTableMapEvent); ok {
```

Comment:
Is it better to add a config for the parser to let the user determine whether to return or ignore this error?